src/backend/aspen/app/views/phylo_trees.py

```diff
@@ -36,6 +36,11 @@ def phylo_trees():
             .options(joinedload(User.group).joinedload(Group.can_see))
             .one()
         )
+        cansee_viewer_group_ids: Set[int] = {
+            cansee.viewer_group_id
```

Comment:
oh, this should be cansee.owner_group_id

Reply:
cool i changed it! Thank you!!